README.md

```diff
@@ -330,6 +330,30 @@ float remapTextureValue(
 
 Copyright © 2018, adidas AG, Released under the [Apache 2.0 License](LICENSE)
 
+## Use Cases
+
+This software can be used to define the visual appearence of virtual materials.
```

Comment:
- `This software`: this is not a software as it is, so, can you explain it choosing a different word?
- If there are several use cases, can you describe them?
- I think instead of `can be used`, we can say directly `is used`.